test-data/unit/check-attr.test

```diff
@@ -1107,3 +1107,47 @@ C(43)
 
 class Yes: ...
 [builtins fixtures/exception.pyi]
+
+[case testTypeInAttrBad]
+import attr
+
+@attr.s
+class C:
+    total = attr.ib(type=Bad)  # E: Name 'Bad' is not defined
+[builtins fixtures/bool.pyi]
+
+[case testTypeInAttrBad2]
```

Comment:
Maybe rename the test case? There seems nothing bad here, just a forward reference.